blog/data/authors/kamil.wcislo.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "Kamil Wcisło",
-    "bio": "",
+    "bio": "Senior Embedded Systems Engineer with 6 years of professional experience. Skilled device driver developer for Linux and firmware in each scale. Dedicated to all wireless communication like LoRa, 802.15.4, WiFi and many other. Former Intel BIOS Software Engineer.",

```

Comment:
@artur-rs can't add suggestions here, it should be over 9 years. On the other side maybe @mek-x would like to edit himself.